code/__DEFINES/mobs.dm

```diff
@@ -161,6 +163,8 @@
 #define HUMAN_MAX_OXYLOSS 3
 #define HUMAN_CRIT_MAX_OXYLOSS (SSMOBS_DT/3)
 
+#define STAMINA_REGEN_BLOCK_TIME (10 SECONDS)
```

Comment:
This was deleted and you reverted it mistakenly

Reply:
breh

Reply:
septic fixed